base/applications/taskmgr/graphctl.c

```diff
@@ -430,34 +430,33 @@ void GraphCtrl_ShiftGrid(TGraphCtrl* this)
 
     solidPen = CreatePen(PS_SOLID, 0, this->m_crGridColor);
 
-    /*  BitBlt it to itself to scroll */
-    BitBlt(this->m_dcGrid, this->m_rectPlot.left, this->m_rectPlot.top + 1,
-           this->m_nPlotWidth, this->m_nPlotHeight, this->m_dcGrid,
-           this->m_rectPlot.left + this->m_nShiftPixels, this->m_rectPlot.top + 1,
+    /*  Scroll the grid left: BitBlt it to itself */
+    BitBlt(this->m_dcGrid, this->m_rectPlot.left, this->m_rectPlot.top+1,
+           this->m_nPlotWidth, this->m_nPlotHeight,
+           this->m_dcGrid, this->m_rectPlot.left + this->m_nShiftPixels, this->m_rectPlot.top+1,
```

Comment:
```suggestion
    BitBlt(this->m_dcGrid, this->m_rectPlot.left, this->m_rectPlot.top + 1,
           this->m_nPlotWidth, this->m_nPlotHeight,
           this->m_dcGrid, this->m_rectPlot.left + this->m_nShiftPixels, this->m_rectPlot.top + 1,
```
Why is `+ 1` needed/correct?